modules/selenium/src/test/java/org/testcontainers/junit/ChromeRecordingWebDriverContainerTest.java

```diff
@@ -59,6 +68,55 @@ public String getFilesystemFriendlyName() {
                 assertEquals("Recorded file not found", 1, files.length);
             }
         }
+
+        @Test
+        public void recordingTestThatShouldHaveCorrectDuration() throws IOException, InterruptedException {
+            File target = vncRecordingDirectory.getRoot();
+            try (
+                // recordAll {
+                // To do this, simply add extra parameters to the rule constructor:
```

Comment:
You can remove those comments. They are our markers for generating code includes in the docs.

```suggestion
```

Reply:
Done, thanks.